packages/documentation-framework/components/example/example.js

```diff
@@ -161,14 +163,17 @@ export const Example = ({
       <div id={previewId} className={css(className, 'pf-v5-u-h-100')}>
         {livePreview}
         {(hasDarkThemeSwitcher || hasRTLSwitcher) && (
-          <Flex direction={{ default: 'column' }} gap={{ default: 'gapLg' }} className="ws-full-page-utils ws-dir-ltr">
+          <Flex direction={{ default: 'column' }} gap={{ default: 'gapLg' }} className="ws-full-page-utils pf-v5-m-dir-ltr ">
```

Comment:
@mcoker is this the only versioned modifier class name?

Reply:
@nicolethoen there are a few others, you can see them here - https://github.com/patternfly/patternfly/blob/825d0af0912528722280460f17c82acf5b66c241/src/patternfly/base/_common.scss

They're global classes that can be applied to anything. Since they aren't paired with a versioned component/layout block or element class, they need to be versioned.